app/views/adventures/new.html.erb

```diff
@@ -47,3 +64,12 @@
     <%= link_to 'Back', adventures_path %>
   </footer>
 </div>
+
+
+<script>
+  window.onload = togglePasswordField(document.getElementById("adventure_has_password"))
+  function togglePasswordField(obj) {
+    var elem = document.getElementsByClassName("Field -passwordField")
+    elem[0].style.display = obj.checked ? 'block' : 'none'
+  }
```

Comment:
is this dynamic enough to show/hide the block when you check the checkbox? I wonder if some heavier JS is required for that, or maybe some clever CSS assembly using `checkbox:checked + Field.-passwordField` (or something like that which is actually valid)

Reply:
It works like that dynamically for me!